lib/tasks/expiration.rake

```diff
@@ -0,0 +1,13 @@
+require 'iso_8601_duration'
+require 'master_rotator'
+require 'installed_rotators'
+
+namespace :expiration do
+  desc "Watch for expired variables and rotate"
+  task :watch, [:account] => [:environment] do |t, args|
+
+    #TODO how to do rake args
+    MasterRotator.new(rotators: InstalledRotators.new, account: args.account)
```

Comment:
Where we'll need to rotate variables across all accounts, would it make more sense to get a variable's account by introspecting the value's variable?